client/account-recovery/reset-password/reset-password-form/test/index.jsx

```diff
@@ -37,32 +38,25 @@ describe( 'ResetPasswordForm', () => {
 		const wrapper = shallow(
 			<ResetPasswordFormComponent
 				resetOptions={ exampleResetOptions }
+				translate={ identity }
 			/>
 		);
 
-		expect( wrapper ).to.have.state( 'isSubmitting' ).to.be.false;
 		expect( wrapper.find( ResetOptionSet ) ).to.have.length( 2 );
-
 		expect( wrapper.find( '.reset-password-form__submit-button' ).prop( 'disabled' ) ).to.be.ok;
 	} );
 
 	context( 'fields', () => {
 		useFakeDom();
 
-		it( 'should be disabled while submitting', function() {
+		it( 'should be disabled while isRequesting is on.', function() {
 			const wrapper = mount(
 				<ResetPasswordFormComponent
 					resetOptions={ exampleResetOptions }
+					isRequesting={ true }
+					translate={ identity }
```

Comment:
I thought the norm here was to default the `translate` prop in the component so we don't need to add it to the tests. Either way I'm cool with it but I think setting the default on the prop will eliminate the duplication from the tests.